api_app/analyzers_manager/observable_analyzers/phishstats.py

```diff
@@ -0,0 +1,41 @@
+import requests
```

Comment:
missing the notice text that is at start of every file.

Reply:
Done